checks/evaluation/branch_protection.go

```diff
@@ -183,49 +151,45 @@ func computeScore(scores []levelScore) (int, error) {
 
 	// First, check if they all pass the basic (admin and non-admin) checks.
 	maxBasicScore := maxScore.basic * len(scores)
-	basicScore := computeNonAdminBasicScore(scores)
-	score += noarmalizeScore(basicScore, maxBasicScore, basicLevel)
-	if basicScore != maxBasicScore {
+	basicScore := sumUpScoreForTier(1, scores, dl)
+	score += normalizeScore(basicScore, maxBasicScore, basicLevel)
+	if basicScore < maxBasicScore {
```

Comment:
whats the difference between the `!=` and `<` here? Can you summarize the approach?

Reply:
For the code execution, it should not make any difference. I changed this because I think `!=` can be a bit confusing for the reader, given that `basicScore` should never be bigger than `maxBasicScore`.

Reply:
> Can you summarize the approach?

Not sure I understood your question. Have I answered it?